src/coreclr/src/jit/codegenarm64.cpp

```diff
@@ -5116,12 +5116,6 @@ void CodeGen::genArm64EmitterUnitTests()
         return;
     }
 
-    if (!compiler->opts.altJit)
-    {
-        // No point doing this in a "real" JIT.
-        return;
-    }
```

Comment:
Perhaps, the beginning and end of `CodeGen::genArm64EmitterUnitTests` should also be guarded with `ifdef ALL_ARM64_EMITTER_UNIT_TESTS` to avoid 
```
*************** In genArm64EmitterUnitTests()
*************** End of genArm64EmitterUnitTests()
```
in the jit dump

Reply:
The "no point" message might have been from before we had much of a "real" JIT. It's fine to remove it.

It would be best if those two messages don't show up in the JIT dump (unless we're actually doing the arm emitter tests)

Reply:
Typically, I would add new tests w/o the ifdef (or with the ifdef commented out), then submit the change with the ifdef enabled. I would (almost) never enable the ifdef -- that's too much spew.

Reply:
Yep, it is almost what I do - but this time I decided to re-check most of the instructions (since I did more extensive changes in the emitter) - so I just defined "global" `ALL_ARM64_EMITTER_UNIT_TESTS`

Reply:
> It would be best if those two messages don't show up in the JIT dump (unless we're actually doing the arm emitter tests)

I will push the change

Reply:
I agree with Bruce here

Reply:
> It would be best if those two messages don't show up in the JIT dump (unless we're actually doing the arm emitter tests)
Pushed the change that guard the print-s with ifdef-endif